src/unsupported.c

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ * Copyright (C) 2023-2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include <umf.h>
+#include <umf/memspace.h>
+
+#if UMF_NO_HWLOC
+
+umf_result_t umfMemspaceCreateFromNumaArray(unsigned *nodeIds, size_t numIds,
```

Comment:
just thinking out loud - should we have a test for that...?

Reply:
We have jobs in our CI that use UMF_DISABLE_HWLOC and use this code.

Reply:
hmm... I'm just saying, this precise function is never called in our tests; we enable memspace numa tests only on Linux when hwloc is enabled - code cov would confirm this 😄 

anyway, I think if this compiles it should be enough, to fix the issue, I guess...